index.js

```diff
@@ -101,14 +112,5 @@ TypeScript.prototype.cacheKeyProcessString = function(string, relativePath) {
 };
 
 TypeScript.prototype.processString = function (string, relativePath) {
-  try{
-    return ts.transpileModule(string, {compilerOptions: this.options, fileName: relativePath}).outputText;
-  }catch(e){
-    console.error('TYPESCRIPT ERROR:');
-    console.error(e.stack + '\n');
-
-    return '';
-  }
```

Comment:
We can explore this further once such an example is demonstrated. Eating all errors though is likely not a good idea.